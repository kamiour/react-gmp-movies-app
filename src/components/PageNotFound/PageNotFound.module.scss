@import 'src/scss/variables';

.pageNotFound {
  padding: 20px;

  .title {
    margin-bottom: 20px;
  }

  .link {
    color: $color_white;

    &:hover {
      color: $color_wild_watermelon;
    }
  }
}