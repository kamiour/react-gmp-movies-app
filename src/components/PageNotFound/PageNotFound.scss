@import 'src/scss/variables';

.page-not-found {
  padding: 20px;

  .title {
    margin-bottom: 20px;
  }

  .link {
    color: $color_white;

    &:hover {
      color: $color_wild_watermelon;
    }
  }
}