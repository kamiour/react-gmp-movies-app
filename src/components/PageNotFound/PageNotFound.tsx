import { Link } from 'react-router-dom';
import './PageNotFound.scss';

const PageNotFound = () => (
  <div className="page-not-found">
    <h1 className="title">Page not found</h1>
    <p>
      <Link className="link" to="/">
        Go to the home page
      </Link>
    </p>
  </div>
);

export default PageNotFound;
