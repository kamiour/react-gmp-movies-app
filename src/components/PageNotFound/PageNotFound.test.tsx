import { render } from '@testing-library/react';
import PageNotFound from './PageNotFound';
import { BrowserRouter as Router } from 'react-router-dom';

describe('PageNotFound', () => {
  it('should match PageNotFound snapshot', () => {
    const { asFragment } = render(
      <Router>
        <PageNotFound />
      </Router>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
