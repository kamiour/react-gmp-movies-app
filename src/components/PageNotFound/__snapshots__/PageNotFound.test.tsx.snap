// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound should match PageNotFound snapshot 1`] = `
<DocumentFragment>
  <div
    class="page-not-found"
  >
    <h1
      class="title"
    >
      Page not found
    </h1>
    <p>
      <a
        class="link"
        href="/"
      >
        Go to the home page
      </a>
    </p>
  </div>
</DocumentFragment>
`;
